ui/src/components/dialog/QasDialog.vue

```diff
@@ -186,6 +186,14 @@ export default {
         useEqualWidth: this.hasAllActions,
         ...this.actionsProps
       }
+    },
+
+    descriptionIsComponent () {
```

Comment:
não seria algo como `hasRenderFunction`?

Reply:
E é certeza que sempre que for componente vai ter esse render?

Reply:
- feito
- de acordo com meus testes sim